ov/power_state.go

```diff
@@ -147,7 +148,7 @@ func (pt *PowerTask) SubmitPowerState(s PowerState) {
 	if s != pt.State {
 		log.Infof("Powering %s server %s for %s.", s, pt.Blade.Name, pt.Blade.SerialNumber)
 		var (
-			body = PowerRequest{PowerState: s.String(), PowerControl: P_MOMPRESS.String()}
+			body = PowerRequest{PowerState: s.String(), PowerControl: P_PRESSANDHOLD.String()}
```

Comment:
this should address https://github.com/HewlettPackard/docker-machine-oneview/issues/86 also